Signup 160453782

lubwamabenja opened the pull request:
This is my sign up page where the user fills in his/her details to create an account
The css files are inside my css branch
please comment
thanks
 @lubwamabenja, @mwinel, @nuwagalb I request your reviews.

lubwamabenja committed:
Add a login page
[Deliveries #160453911]

lubwamabenja committed:
Add a signup page
[Deliveries #160453782]

mwinel commented:
All the code looks ok though not well organised which makes it harder for reviewers to follow through. Also try to include a way to test this code in your PR.

nuwagalb commented on html/signup.html:
```diff
@@ -0,0 +1,28 @@
+			<input type="password" name="" placeholder=" Enter password">
+			<input type="submit" name="" value="Signup">
+			<a href="index.html">Login?</a><br>
+
```
this extra space is unnecessary. I would suggest that you remove it

nuwagalb reviewed (commented):
I cannot see your css code. Have you added styling to the page?

lubwamabenja commented:
In my css folder

On Tue, 18 Sep 2018, 12:35 nuwagalb, <notifications@github.com> wrote:

> *@nuwagalb* commented on this pull request.
>
> I cannot see your css code. Have you added styling to the page?
> ------------------------------
>
> In html/signup.html
> <https://github.com/lubwamabenja/TODOLIST/pull/1#discussion_r218361526>:
>
> > +	<link rel="stylesheet" type="text/css" href="../css/login.css">
> +</head>
> +<body>
> +	<div class="loginbox">
> +		
> +		<h1>SignUp Here</h1>
> +		<form enctype="">
> +			<p>Username</p>
> +			<input type="text" name="" placeholder="Enter Username">
> +			<p>Email</p>
> +			<input type="Email" name="" placeholder="Enter Email address">
> +			<p>Password</p>
> +			<input type="password" name="" placeholder=" Enter password">
> +			<input type="submit" name="" value="Signup">
> +			<a href="index.html">Login?</a><br>
> +
>
> this extra space is unnecessary. I would suggest that you remove it
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/lubwamabenja/TODOLIST/pull/1#pullrequestreview-156279280>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ApLCTZva1mkBsDusAegdkSX6q-NiOCxDks5ucL5RgaJpZM4Wshci>
> .
>

lubwamabenja committed:
feature(signup-160453782) Adding stylesheet to the signup page
This commit adds styling t the signup page and the background image
[Deliveries #160453782]
